docker/build-container.sh

```diff
@@ -3,12 +3,37 @@
 export UID=$(id -u)
 export GID=$(id -g)
 
-for arg in $@; do
-    if [[ $arg == clean ]]; then
-        docker rm cv-hardware-parts
-	docker rmi ia-bigdata
-    fi;
+while [[ $# -gt 0 ]]; do
+    case $1 in 
+    --clean-container | --clean-all)
+        docker rm ai-palatal-rugoscopy-env > /dev/null 2>&1
+        ;;
+    --clean-image | --clean-all)
+        docker rmi ai-palatal-rugoscopy > /dev/null 2>&1
```

Comment:
`ai-palatal-rugoscopy` and `ai-palatal-rugoscopy-env` are used several times in the script. Make them constants.